test/integration/ComputeFlow.test.ts

```diff
@@ -21,10 +20,12 @@ const data = JSON.parse(
   )
 )
 
+const config = configHelperNetworks[1]
```

Comment:
Please use getConfig (https://github.com/oceanprotocol/ocean.js/blob/v4main/src/utils/ConfigHelper.ts#L205)  with web3.chainId to get the correct network.
It's not safe to assume that configHelperNetworks[1] will always be development